src/com/company/Main.java

```diff
@@ -1,8 +1,184 @@
 package com.company;
```

Comment:
Please use the right package name. 

com.company can be changed to com.hotwaxsystems or com.hotwaxsystems.www.